ambry-router/src/test/java/com/github/ambry/router/GetBlobOperationTest.java

```diff
@@ -1315,18 +1317,20 @@ public void testRouterMetadataCacheFirstChunkOperationFlag() throws Exception {
         .build(), false, routerMetrics.ageAtGet);
     // First range request gets the entire blob including blobInfo
     GetBlobOperation op1 = getAndAssertSuccess(false, false, (short) 0);
-    assertTrue(String.format("Expected operation flag %s but received %s", MessageFormatFlags.All,
-        op1.getFirstChunkOperationFlag()), op1.getFirstChunkOperationFlag() == MessageFormatFlags.All);
+    assertSame(String.format("Expected operation flag %s but received %s", MessageFormatFlags.All,
+        op1.getFirstChunkOperationFlag()), op1.getFirstChunkOperationFlag(), MessageFormatFlags.All);
```

Comment:
You are comparing op1.getFirstChunkOperationFlag() with op1.getFirstChunkOperationFlag() instead of op2.getFirstChunkOperationFlag().
You should use assertEquals. since assertEquals() Asserts that two objects are equal. assertSame() Asserts that two objects refer to the same object.

Reply:
done

Reply:
done fixed